docs/tutorials/gluon/multi_gpu.md

```diff
@@ -0,0 +1,194 @@
+<!--- Licensed to the Apache Software Foundation (ASF) under one -->
+<!--- or more contributor license agreements.  See the NOTICE file -->
+<!--- distributed with this work for additional information -->
+<!--- regarding copyright ownership.  The ASF licenses this file -->
+<!--- to you under the Apache License, Version 2.0 (the -->
+<!--- "License"); you may not use this file except in compliance -->
+<!--- with the License.  You may obtain a copy of the License at -->
+
+<!---   http://www.apache.org/licenses/LICENSE-2.0 -->
+
+<!--- Unless required by applicable law or agreed to in writing, -->
+<!--- software distributed under the License is distributed on an -->
+<!--- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY -->
+<!--- KIND, either express or implied.  See the License for the -->
+<!--- specific language governing permissions and limitations -->
+<!--- under the License. -->
+
+# Multiple GPUs training with Gluon API
+
+In this tutorial we will walk through how one can train deep learning neural networks on multiple GPUs within a single machine. This tutorial focuses on data parallelism oppose to model parallelism. The latter is not supported by Apache MXNet out of the box, and one have to manually route the data among different devices to achieve model parallelism. Check out [model parallelism tutorial](https://mxnet.incubator.apache.org/versions/master/faq/model_parallel_lstm.html) to learn more about it.
+Here we will focus on implementing data parallel training for a convolutional neural network LeNet.
+
+## Prerequisites
+
+- Two or more GPUs 
+- Cuda 9 or higher
+- CUDNN v7 or higher
+- Knowledge of how to train a model using Gluon API
+
+## Storing data on GPU
+
+The basic primitive in Apache MXNet to specify a tensor is [NDArray](https://mxnet.incubator.apache.org/api/python/ndarray/sparse.html#module-mxnet.ndarray). When one creates NDArray it has to provide the context - a place where this tensor is going to be stored. The context can be either CPU or GPU and both can be indexed: if your machine has multiple GPUs, you can provide an index to specify which GPU to use. By default, CPU context is used, and that means that the tensor will live in main RAM. Below is an example how to create two tensors where one is stored on the first GPU and the second is stored on the second GPU.
+
+```python
+import mxnet as mx
+
+a = mx.nd.array([1, 2, 3], ctx=mx.gpu(0))
+b = mx.nd.array([5, 6, 7], ctx=mx.gpu(1))
```

Comment:
the tutorial nightly test has changed to use P3.2xlarge with 1GPU, so this may fail

Reply:
Hm, whole this tutorial is about how to do multigpu training. I guess if this is the case, I will have to remove it from nightly tests.

Reply:
yes, maybe comment and add it to whitelist: https://github.com/apache/incubator-mxnet/blob/master/tests/tutorials/test_sanity_tutorials.py#L27

Reply:
Good idea, added to the whitelist

Reply:
I don't think it's a good idea to not test it. I'll suggest changes to make it testable and carry the same information